lib/db/mysql.js

```diff
@@ -674,13 +675,13 @@ module.exports = function (log, error) {
     )
   }
 
-  // Update : accounts
-  // Set    : emailVerified = true
-  // Where  : uid = $1
-  var VERIFY_EMAIL = 'CALL verifyEmail_3(?)'
+  // Update : accounts, emails
+  // Set    : emailVerified = true if email is in accounts table or isVerified = true if on email table
+  // Where  : uid = $1, emailCode = $2
+  var VERIFY_EMAIL = 'CALL verifyEmail_4(?, ?)'
```

Comment:
Not sure of the historical significance but, `verifyEmail` did not actually look and verify that the `emailCode` belonged to the user account. It simply verified the users email. Which made me wonder, what was the point of the `emailCode` originally?